docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -209,6 +209,8 @@ p.readFrom(source)
 The key and value functions have certain limitations. They can be used to map only to objects which the Hazelcast member can deserialize, which unfortunately doesn't include user code submitted as a part of the job. So in the above example it's OK to have `String` email values, but we wouldn't be able to use `Customer` directly.
 
 If user code has to be used, then the problem can be solved with the help of the User Code Deployment feature. Example configs for that can be seen in our xref:pipelines:cdc-join.adoc#7-start-hazelcast-jet[CDC Join tutorial].
+
+Although User Code Deployment has been deprecated, the replacement User Code Namespaces feature does not yet support Jet jobs or pipelines. For now, continue to use the User Code Deployment solution in this scenario.
```

Comment:
> User Code Namespaces feature does not yet support Jet jobs

@JamesHazelcast is this accurate after https://github.com/hazelcast/hazelcast-mono/pull/3293?

Reply:
Part of the work for UCN support in Jet is to update documentation: https://hazelcast.atlassian.net/browse/HZG-88

The work is currently in QA